docs/spark-standalone.md

```diff
@@ -93,6 +93,10 @@ You can optionally configure the cluster further by setting environment variable
 <table class="table">
   <tr><th style="width:21%">Environment Variable</th><th>Meaning</th></tr>
   <tr>
+    <td><code>SPARK_MASTER</code></td>
```

Comment:
Isn't this deprecated though?  I don't know that this is supposed to be documented.